artworks/aitagger.go

```diff
@@ -0,0 +1,37 @@
+package artworks
+
+import (
+	"strings"
+)
+
+type AITagger struct{}
```

Comment:
There's no reason for this to be a struct, it can just be a function in artworks package called IsAI or something similar.